### PR TITLE
Document how to select OVN with KIND

### PR DESCRIPTION
**Pull request by sridhargaddam**
Signed-Off-by: Sridhar Gaddam <sgaddam@redhat.com>

### CODE REVIEW

File: src/content/development/building-testing/_index.en.md [hunk 1]
```diff
@@ -81,6 +81,12 @@ make e2e using=helm,globalnet
 
 The [cable driver](../../getting-started/architecture/gateway-engine) used to connect clusters can also be selected with `using` flags:
 
+```shell
+make e2e using=vxlan
```
Comment (dfarrell07) [inline src/content/development/building-testing/_index.en.md:85]: I worry a bit about providing VXLAN as an example without noting that it shouldn't be used on unencrypted connections.

File: src/content/development/building-testing/_index.en.md [hunk 2]
```diff
@@ -81,6 +81,12 @@ make e2e using=helm,globalnet
...
+```
+
+In order to deploy clusters with OVN-Kubernetes, the following command can be used:
```
Comment (nyechiel) [inline src/content/development/building-testing/_index.en.md]:
```suggestion
In order to deploy clusters with [OVN Kubernetes](https://submariner.io/getting-started/architecture/networkplugin-syncer/ovn-kubernetes/), the following command can be used:
```

File: src/content/development/building-testing/_index.en.md (hunk 1 above)
Comment (nyechiel) [reply src/content/development/building-testing/_index.en.md:85]: Alternatively, we can use wireguard, but IMHO it's fine to mention VXLAN here. In the end, this is only a local build/test environment.

**Review by nyechiel — CHANGES_REQUESTED** [review]

File: src/content/development/building-testing/_index.en.md (hunk 1 above)
Comment (nyechiel) [reply src/content/development/building-testing/_index.en.md:85]: And FWIW, we have this pending [issue](https://github.com/submariner-io/submariner-website/issues/597) to better clarify how/when VXLAN should be used.

**Review by nyechiel — APPROVED** [review]
LGTM, thanks @sridhargaddam

**Review by Jaanki — APPROVED** [review]

**Review by dfarrell07 — APPROVED** [review]

**Pull request merged**